Source/Eto.Test/Eto.Test/Sections/Controls/DrawableSection.cs

```diff
@@ -9,23 +9,58 @@ public class DrawableSection : Scrollable
 	{
 		public DrawableSection()
 		{
-			var layout = new DynamicLayout { DefaultSpacing = new Size(5, 5), Padding = new Padding(10) };
-
-			layout.BeginVertical();
-			layout.BeginHorizontal();
-			layout.Add("Default");
-			layout.Add(this.Default(), xscale: true);
-			layout.Add("With Background");
-			layout.Add(this.WithBackground(), xscale: true);
-			layout.EndHorizontal();
-			layout.EndVertical();
-			layout.BeginVertical();
-			// use a separate containing panel to test calculations in those cases
-			layout.AddRow("Large Canvas", new Panel { Content = this.LargeCanvas() });
-			layout.EndVertical();
-
-			layout.Add(null);
-			Content = layout;
+			Content = new TableLayout
+			{
+				Padding = new Padding(10),
+				Spacing = new Size(10,10),
+				Rows =
+				{
+					TableLayout.HorizontalScaled(
+						10,
+						new TableLayout(
+							"Default",
+							Default()
+						),
+						new TableLayout(
+							"With Background",
+							WithBackground()
+						)
+					),
+
+					new TableLayout(
+						"Large Canvas",
+						// use a separate containing panel to test calculations in those cases
+						new Panel { Content = LargeCanvas() }
+					),
+
+					new TableRow(TableLayout.Horizontal(
+						10,
+						new TableLayout(
+							"Nested",
+							Nested()
+						),
+						new TableLayout(
+							"Transparent",
+							Transparent()
+						),
+						new TableLayout(
+							"Tools",
+							TableLayout.Horizontal(
+								Tools(1), Tools(2), Tools(0)
+							),
+							Tools(3),
+							Tools(0)
+						)
+					)),
+
+					(Platform.SupportedFeatures & PlatformFeatureFlags.DrawableWithTransparentContent) == 0 ?
+					new TableRow(
+						"(Transparent content on drawable not supported on this platform)"
+					) : null,
+
+					null
+				}
+			};
```

Comment:
I really needed this to test it properly and thus added `TableLayout.Horizontal[Scaled]` and few more.